ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorTest.java

```diff
@@ -439,17 +440,21 @@ public void shouldRunSetStatements() {
     standaloneExecutor.start();
 
     // Then:
-    verify(ksqlEngine).execute(eq(ConfiguredStatement.of(cs, ImmutableMap.of("name", "value"), ksqlConfig)));
+    verify(ksqlEngine).execute(
+        eq(ConfiguredStatement.of(
```

Comment:
nit: `eq` is unnecessary. 

I realize you didn't add it, but would you remove it please?